syntax = "proto3";

service AuthService {
  rpc Login(LoginRequest) returns (AuthResponse);
  rpc Logout(LogoutRequest) returns (AuthResponse);
}

message LoginRequest {
  string username = 1;
  string password = 2;
}

message LogoutRequest {
  string session_id = 1;
}

message AuthResponse {
  bool success = 1;
  string message = 2;
  string session_id = 3;
}